lib/rules/match-component-file-name.js

```diff
@@ -102,19 +102,25 @@ module.exports = {
      */
     function verifyName(node) {
       let name
+      let quote = ''
       if (node.type === 'TemplateLiteral') {
         const quasis = node.quasis[0]
         name = quasis.value.cooked
+        quote = '`'
       } else {
         name = `${node.value}`
+        quote = node.raw[0]
       }
 
       if (!compareNames(name, filename)) {
         errors.push({
           node,
           message:
             'Component name `{{name}}` should match file name `{{filename}}`.',
-          data: { filename, name }
+          data: { filename, name },
+          fix(fixer) {
+            return fixer.replaceText(node, `${quote}${filename}${quote}`)
+          }
```

Comment:
I think this should not be an auto-fix, since it's sometimes the case that the filename is wrong and the name property is correct.

Instead, please change this to be a [suggestion](https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions) instead. Then cases where the filename is correct and the name property is wrong can be easily corrected in the editor, without causing wrong fixes when running from the commandline.

Reply:
Thank you very much. I understand the reason to using suggest instead of fix.